app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -73,69 +79,49 @@ class InstallerViewModel(
     private var inputFile: File? = null
 
     private var installedApp: InstalledApp? = null
-    var isInstalling by mutableStateOf(false)
+    internal var isInstalling by mutableStateOf(false)
         private set
-    var installedPackageName by mutableStateOf<String?>(null)
+    internal var installedPackageName by mutableStateOf<String?>(null)
```

Comment:
```suggestion
    var installedPackageName by mutableStateOf<String?>(null)
```
Is internal necessary?